docs/install_guides/installing-model-dev-tools.md

```diff
@@ -21,12 +21,28 @@ If you are a Python developer, follow the steps in the <a href="openvino_docs_in
 ## <a name="cpp-developers"></a>For C++ Developers
 If you are a C++ developer, you must first install OpenVINO Runtime separately to set up the C++ libraries, sample code, and dependencies for building applications with OpenVINO. These files are not included with the PyPI distribution. See the [Install OpenVINO Runtime](./installing-openvino-runtime.md) page to install OpenVINO Runtime from an archive file for your operating system.
 
-Once OpenVINO Runtime is installed, you may install OpenVINO Development Tools for access to tools like Model Optimizer, Model Downloader, Benchmark Tool, and other utilities that will help you optimize your model and develop your application. Follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it. In Step 4, make sure that you follow the instructions in the "C++" tab.
+Once OpenVINO Runtime is installed, you may install OpenVINO Development Tools for access to tools like Model Optimizer, Model Downloader, Benchmark Tool, and other utilities that will help you optimize your model and develop your application. Follow the steps in the <a href="#install-dev-tools">Installing OpenVINO Development Tools</a> section on this page to install it.
 
 ## <a name="install-dev-tools"></a>Installing OpenVINO™ Development Tools
 Follow these step-by-step instructions to install OpenVINO Development Tools on your computer.
+There are two options to install OpenVINO Development Tools: installation into an existing environment with a deep learning framework that was used
+for model training or creation; or installation into a new environment.
 
-### Step 1. Set Up Python Virtual Environment
+### Installation into an Existing Environment with the Source Deep Learning Framework
+
+To install OpenVINO Development Tools (see the [What's in the Package](#whats-in-the-package) section of this article) into an existing environment
+with the deep learning framework used for the model training or creation, run the following command:
+
+```sh
+pip install openvino-dev
+```
+
+### Installation in a New Environment
+
+If you do not have an environment with a deep learning framework for the input model or you encounter any compatibility issues between OpenVINO
+and your deep learning framework, you may install OpenVINO Development Tools with validated versions of frameworks into a new environment. 
```

Comment:
I would clarify this as "compatibility issues between openvino and your VERSION of deep learning framework"

Reply:
Done.